bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/PrefixFilterAdapter.java

```diff
@@ -53,4 +52,46 @@ public FilterSupportStatus isFilterSupported(
       FilterAdapterContext context, PrefixFilter filter) {
     return FilterSupportStatus.SUPPORTED;
   }
+
```

Comment:
Is there a reason to put this method here vs. BigtableExtendedScan?  This seems like a utility method, and perhaps should be in com.google.cloud.bigtable.util.